src/fastoad_cs25/models/variable_descriptions.txt

```diff
@@ -156,6 +156,8 @@ data:geometry:wing:MAC:y || Y-position of mean aerodynamic chord of wing
 data:geometry:wing:area || wing reference area
 data:geometry:wing:aspect_ratio || wing aspect ratio
 data:geometry:wing:b_50 || actual length between root and tip along 50% of chord
+data:geometry:wing:center:chord || chord length at wing Y axis of symmetry
```

Comment:
I fear this might not be clear enough maybe add a small precision
```suggestion
data:geometry:wing:center:chord || chord length at wing Y axis of symmetry (fuselage center line)
```